### ISSUE TITLE
Graph todos

### ISSUE THREAD

**Issue by slowbackspace** [trezor/trezor-suite#1613]
- [x] refactor fetch logic
  - [x] fetch on account create (bonus points if it starts only after discovery process is complete) and, until caching is implemented, fetch also on loading accounts from storage. Alternative could be fetching on discovery.complete => fetch only new accounts
  - [x] prune data for disabled networks (removed accounts)
  - [x] update graph data on new transactions
  - [x] fix fetching data with remembered disconnected device

- [x] persistent storage for graph data

- [x] UI enhancements
   - [x] paddings (around last bar on x axis)
   - [x] labels on Y (linear scale, domain [-max, max])
   - [x] responsivity

- [x] performance experiments
  - [x] disabling animation (not much changed, keeping them enabled for now)
  - [x] optimalize rerenders
  - [x] web worker - data processing for all/year chart takes too long (around 2s until the user can access dashboard)

- [x] exclude sent to self txs amounts (waiting for blockbook implementation) https://github.com/trezor/trezor-suite/issues/1779

still todo, but low priority
- [ ] optimizations: already downloaded data stays valid, we just need to fetch additional data using from, to timestamps, maybe prune older data that no longer fit in selected range or filter based on timestamps. When switching range consider only using available data (refetch could be still triggered with refresh button) OR just fetch additional timestamps, not full range

feedback
- https://github.com/trezor/trezor-suite/issues/1464#issuecomment-617262936
- https://github.com/trezor/trezor-suite/issues/1074#issuecomment-614757826
- https://github.com/trezor/trezor-suite/issues/833#issuecomment-614737022

**Comment by bosomt** [comment]
I don't know if this is correct issue,
but I think that info that i received 2000$+ is really confusing
First I thought that it's some kind of bug.

I went thru all transactions 02/2020 and only after some time i found out that it means summ of whole month.

![image](https://user-images.githubusercontent.com/31506317/81082750-fa8d9000-8ef3-11ea-8442-03e4184b6d3f.png)

same account in wallet: 
no info about 0.3 BTC i received, received ballance is 0.03 BTC

![image](https://user-images.githubusercontent.com/31506317/81082933-37598700-8ef4-11ea-8aac-234431ac9469.png)
